docs/static/security/es-security.asciidoc

```diff
@@ -0,0 +1,76 @@
+[discrete]
+[[es-security-on]]
+== {es} security on by default
+
+{es} {ref}/configuring-stack-security.html[security is on by default] starting in 8.0. 
+{es} is the certificate authority (CA), and it generates its own self-signed Secure Sockets Layer (SSL) certificate at startup. 
+{ls} must establish a secure connection using the SSL certificate before it can transfer data. 
```

Comment:
```suggestion
For self-hosted {es} cluter, {ls} must establish a secure connection using the self-signed SSL certificate before it can transfer data. 
```